lib/cc/cli/init.rb

```diff
@@ -9,6 +9,7 @@ class Init < Command
 
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
+          create_default_configs
           fatal "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
```

Comment:
> What was our original reason in favor of going with the exitcode 1 when there's no .codeclimate.yml

The logic predated me, but it was failing with exit code 1 if `.codeclimate.yml` **already existed** because `init`'s primary purpose was to create that file.

> What would you think about something more neutral like .codeclimate.yml already present, skipping generation

I think that sounds fine (`#warn` seems appropriate for the output). The verb "generation" would not be appropriate for the `--upgrade` case, though: it seems like you'll need another `*_verb` method on the generator class to distinguish between the generated/upgraded.